config/routes.rb

```diff
@@ -4,12 +4,14 @@
   resources :categories, only: [:index, :show], param: :slug
   resources :cart_stickers, only: [:create, :index, :destroy]
   resources :cart, only: [:index, :create, :destroy]
-  resources :users, only: [:new, :create]
+  resources :users, only: [:new, :create, :update]
   resources :orders, only: [:create, :index, :show]
   get '/login', to: "session#new"
   post '/login', to: "session#create"
   delete '/logout', to: "session#destroy"
   get '/dashboard', to: "users#show"
+  get '/dashboard/edit', to: "users#edit"
+  put '/dashboard', to: "users#update"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.